### PR TITLE
Remove allowFileUploads ACP option

### PR DESCRIPTION
**Pull request by julianlam**

### CODE REVIEW

File: src/upgrade.js [hunk 1]
```diff
@@ -152,35 +153,34 @@ Upgrade.process = function (files, skipCount, callback) {
...
+				process.stdout.write('  → '.white + String('[' + [date.getUTCFullYear(), date.getUTCMonth() + 1, date.getUTCDate()].join('/') + '] ').gray + String(scriptExport.name).reset + '...');
 
 				// For backwards compatibility, cross-reference with schemaDate (if found). If a script's date is older, skip it
 				if ((!results.schemaDate && !results.schemaLogCount) || (scriptExport.timestamp <= results.schemaDate && semver.lt(version, '1.5.0'))) {
-					readline.clearLine(process.stdout, 0);
-					readline.cursorTo(process.stdout, 0);
-					readline.moveCursor(process.stdout, 0, -1);
-					console.log('  → '.white + String('[' + [date.getUTCFullYear(), date.getUTCMonth() + 1, date.getUTCDate()].join('/') + '] ').gray + String(scriptExport.name).reset + '... ' + 'skipped'.grey);
+					process.stdout.write(' skipped\n'.grey);
 					db.sortedSetAdd('schemaLog', Date.now(), path.basename(file, '.js'), next);
```
Comment (barisusakli) [inline src/upgrade.js:161]: there is no `next` anymore

File: src/upgrade.js [hunk 2]
```diff
@@ -152,35 +153,34 @@ Upgrade.process = function (files, skipCount, callback) {
...
-					readline.cursorTo(process.stdout, 0);
-					readline.moveCursor(process.stdout, 0, -1);
-					console.log('  → '.white + String('[' + [date.getUTCFullYear(), date.getUTCMonth() + 1, date.getUTCDate()].join('/') + '] ').gray + String(scriptExport.name).reset + '... ' + 'OK'.green);
+				process.stdout.write(' OK\n'.green);
 
-					// Record success in schemaLog
-					db.sortedSetAdd('schemaLog', Date.now(), path.basename(file, '.js'), next);
-				});
+				// Record success in schemaLog
+				db.sortedSetAdd('schemaLog', Date.now(), path.basename(file, '.js'));
```
Comment (barisusakli) [inline src/upgrade.js:183]: missing await

File: src/upgrades/1.13.4/remove_allowFileUploads_priv.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const db = require('../../database');
+const privileges = require('../../privileges');
+
+module.exports = {
+	name: 'Removing file upload privilege if file uploads were disabled (`allowFileUploads`)',
+	timestamp: Date.UTC(2020, 4, 21),
+	method: async () => {
+		const allowFileUploads = parseInt(await db.getObjectField('config', 'allowFileUploads'), 10);
+		if (allowFileUploads === 1) {
+			return;
```
Comment (barisusakli) [inline src/upgrades/1.13.4/remove_allowFileUploads_priv.js:12]: Should probably call `await db.deleteObjectField('config', 'allowFileUploads');`  or the field will stay in db after upgrade

File: src/upgrade.js (hunk 1 above)
Comment (julianlam) [reply src/upgrade.js:161]: Wasn't planning on changing the underlying code, so this line was left alone :grimacing:

File: src/upgrade.js (hunk 2 above)
Comment (julianlam) [reply src/upgrade.js:183]: Same with this line :thinking:

File: src/upgrades/1.13.4/remove_allowFileUploads_priv.js (hunk 3 above)
Comment (julianlam) [reply src/upgrades/1.13.4/remove_allowFileUploads_priv.js:12]: I do this at the end of the file :thinking: unless I'm misunderstanding?

Comment (julianlam) [reply src/upgrades/1.13.4/remove_allowFileUploads_priv.js:12]: oh, I am misunderstanding.

File: src/upgrade.js (hunk 2 above)
Comment (julianlam) [reply src/upgrade.js]: Nevermind, got it :+1:

**Pull request merged**